ScrutinyPlot/mkscrutinydatafiles.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+datadir=/afs/cern.ch/work/c/cvuosalo/popularity/
+wkdir=/tmp/cvuosalo/popularity
+# 1. Get datasets and their sizes
+# Fields 2, 7, and 6 are dataset name, average size, and end date of its presence
+awk -F , '{print $2 "," $7 "," $6}' $datadir/ecomjobdata/phedex.csv | grep -v 'dataset,ave' | sort -t , -k 1,1 -u >> $wkdir/dsandsz$$.txt
```

Comment:
Here again you rely on hardcoded structure $datadir/ecomjobdata which may not be the case in general. Please change code to use $datadir.